core/src/main/scala/kafka/tools/GetOffsetShell.scala

```diff
@@ -164,20 +167,27 @@ object GetOffsetShell {
   }
 
   /**
-   * Creates a topic-partition filter based on a list of patterns.
+   * Creates a topic filter and a topic-partition filter based on a list of patterns.
    * Expected format:
    * List: TopicPartitionPattern(, TopicPartitionPattern)*
    * TopicPartitionPattern: TopicPattern(:PartitionPattern)? | :PartitionPattern
    * TopicPattern: REGEX
    * PartitionPattern: NUMBER | NUMBER-(NUMBER)? | -NUMBER
    */
-  def createTopicPartitionFilterWithPatternList(topicPartitions: String, excludeInternalTopics: Boolean): PartitionInfo => Boolean = {
+  def createTopicPartitionFilterWithPatternList(topicPartitions: String,
+                                                excludeInternalTopics: Boolean
+                                               ): (String => Boolean, PartitionInfo => Boolean) = {
```

Comment:
nit: Could we format methods as follow? We tend to follow this pattern nowadays.

```scala
def createTopicPartitionFilterWithPatternList(
  topicPartitions: String,
  excludeInternalTopics: Boolean
): (String => Boolean, PartitionInfo => Boolean) = {

}
```